ansible/config_sonic_basedon_testbed.yml

```diff
@@ -334,4 +334,10 @@
         become: true
         shell: config save -y
         when: save is defined and save|bool == true
+
+      - name: cleanup cached facts
+        shell: python ../tests/common/cache/facts_cache.py {{ inventory_hostname }}
+        delegate_to: localhost
```

Comment:
If the task is delegated to ```localhost```, what's the value of ```inventory_hostname``` ? Is it ```localhost``` or the hostname of DUT? I'm not quite sure.

Reply:
It's the hostname of DUT. I've tested that.

Reply:
Got it. Thanks.